src/components/AvatarWithImagePicker.js

```diff
@@ -194,7 +195,12 @@ class AvatarWithImagePicker extends React.Component {
                     return;
                 }
 
-                this.setState({isAvatarCropModalOpen: true, imageUri: image.uri, imageName: image.name});
+                this.setState({
+                    isAvatarCropModalOpen: true,
+                    imageUri: image.uri,
+                    imageName: image.name,
+                    imageType: image.type,
```

Comment:
FYI this (image type) is the main "new thing" that we're passing down to `cropOrRotateImage`